### PR TITLE
Release 0.8.8

### PR DESCRIPTION
**Pull request by brucemiller**
Various bits corresponding to release 0.8.8.

ready for cpantesters, etc.

@dginev  please check mods to testing (there were unnecessary warnings during make test on some platforms)

### CODE REVIEW

**Comment by brucemiller** [comment]
@dginev and please check what the test fail; something about docker login??

**Comment by dginev** [comment]
The docker bits appear to be some github CI instability - will double-check. I will try rerunning CI until "it clicks" :> 

For the warning you patched - the change is reasonable, and I think I spotted it once as well. But it kind of tests a failure case - `-s $file` must fail, i.e. the file is not found on disk? Your patch is fine, irrespective.

File: Changes [hunk 1]
```diff
@@ -1,3 +1,49 @@
+0.8.8 2024-02-29
+      - This release addresses a large variety of usability, fidelity, robustness,
+        portability and output-quality issues.
+      - Improved CSS, html, accessibility affecting
+        * framing, position & rotation, color
+        * figure/subfigure grouping
+      -  NathML (closer to MathML Core)
```
Comment (dginev) [inline Changes:7]: That's a fun typo, let me mark it while I think of a joke.

**Comment by dginev** [comment]
Should we thank the Math WG for help with fleshing out new related features, such as #2292 and #2303 ? 

Fred, Neil and David in particular have been quite helpful.

**Review by dginev — APPROVED** [review]
v0.8.8 approved to merge 🚀

**Pull request merged**